src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java

```diff
@@ -70,7 +70,6 @@
         tmp.add("session_id");
         tmp.add("state");
         tmp.add("sync_id");
-        tmp.add("emoji");
```

Comment:
Why remove this again? Does the Rich presence for Custom status have one of the other fields at least?

Reply:
We no longer treat it as rich presence

Reply:
Hm? So currently, a normal Activity is created, but the activity itself doesn't hold the correct text or the emoji yet, right?

The emoji would be given to the RichPresence Constructor, but that one is only called, when one of the rich-presence flags is set